google/cloud/dataproc_spark_connect/session.py

```diff
@@ -407,9 +407,11 @@ def _get_dataproc_config(self):
                     )
                 }
             if "COLAB_NOTEBOOK_ID" in os.environ:
-                dataproc_config.labels["colab-notebook-id"] = os.environ[
-                    "COLAB_NOTEBOOK_ID"
-                ]
+                colab_notebook_id_full = os.environ["COLAB_NOTEBOOK_ID"]
+                # Extract the last part of the path, which is the ID
+                dataproc_config.labels["colab-notebook-id"] = os.path.basename(
```

Comment:
I think that this assumes that notebook is always in the same project and region as session, but we can not make this assumption because users can create session in any project and region from the notebook